main/resource-access.bs

```diff
@@ -130,31 +125,150 @@ Note:
+component of `PUT`, `POST` and `PATCH` requests.
+[[Source](https://github.com/solid/specification/issues/68#issuecomment-561690124)]
+
+Servers MUST allow creating new resources with a `POST` request to URI path
 ending `/`. Servers MUST create a resource with URI path ending `/{id}` in
 container `/`. Servers MUST create a container with URI path ending `/{id}/`
 in container `/` for requests including the HTTP `Link` header with
 `rel="type"` targeting a valid LDP container type. Servers MUST handle
 subsequent requests to the newly created container's URI as if it is a valid
-LDP container type by including it the HTTP response's `Link` header.
+LDP container type by including the HTTP response's `Link` header.
+[[Source](https://github.com/solid/specification/pull/160#issuecomment-636822687)]
+
+When a `POST` method request targets a non-container resource without an
+existing representation, the server `MUST` respond with the `404` status code.
+[[Source](https://github.com/solid/specification/issues/108#issuecomment-549448159)]
+
+When a `PUT` or `PATCH` method request targets an auxiliary resource, the
+server MUST create or update it. When a `POST` method request with the `Slug`
+header targets an auxiliary resource, the server MUST respond with the `403`
+status code and response body describing the error.
+[[Source](https://github.com/solid/specification/issues/42#issuecomment-616688848)]
+
+Servers SHOULD NOT allow HTTP `POST`, `PUT` and `PATCH` to update a
+container's containment triples; if the server receives such a request, it
+SHOULD respond with a `409` status code.
+[[Source](https://github.com/solid/specification/issues/40#issuecomment-573358652)]
+
+Clients MAY use the HTTP `If-None-Match` header with a value of `"*"` to
+prevent an unsafe request method (eg. `PUT`, `PATCH`) from inadvertently
+modifying an existing representation of the target resource when the client
+believes that the resource does not have a current representation.
+[[Source](https://github.com/solid/specification/issues/108#issuecomment-567272797)]
+[[Source](https://github.com/solid/specification/issues/40#issuecomment-566995240)]
+
+When using Web Access Control ([[#wac]]):
+
+To create or update an ACL resource (see [[#ar-wac]]), an `acl:agent` MUST
+have `acl:Control` privileges per the ACL inheritance algorithm on the
+resource directly associated with it.
+[[Source](https://github.com/solid/specification/issues/42#issuecomment-616688848)]
+
+### Deleting Resources ### {#delete}
+
+When a server supports the HTTP `DELETE` method [[!RFC7231]] this
+specification imposes the following requirements:
+[[Source](https://github.com/solid/specification/issues/39#issuecomment-538017667)]
+
+When a `DELETE` request targets storage's root container or its associated ACL
+resource, the server MUST respond with the `405` status code. Server MUST
+exclude the `DELETE` method in the HTTP response header `Allow` in response to
```

Comment:
Just to clarify: does this mean that, in order to delete one's Pod, there needs to be some separate (implementation specific?) mechanism for deleting the root container?

Reply:
Yes. Based on the available information, I'm inclined to say that how the root container come into existence (or commissioned) and cease to exist is unknown. It is an acceptable axiom. [The observable universe is after the big bang and just before it collapses? - /me pokes @kjetilk ]

Prohibiting root container or its ACL from getting deleted was to partly address loss of control. We could of course allow an authorized agent to delete the root container (provided that the container is empty), and subsequent requests to read the root container may get a 404/405/410. Practically speaking, I'm not sure if the difference between an empty root (and no public read) and one that does not really exist is significant. Even once deleted, there'd be no particular way to reinstate the root container given that the agent that was previously authorized is presumably no longer known to the system.

An alternative is to have an agent with the Control access (with the role of admin/controller/authority/creator..) on the root container where the authorization policy is sticky, immutable or maintains a specific shape and data. "But wait!" you say, how did that information come to be?

So, we are back again to a separate mechanism that somehow commissions the root container/storage as well as being able to decommission. We could specify an out of band management mechanism but that'd still be orthogonal.

Open to ideas / implementation experience that can give us a more useful axiom to work with.

Reply:
re: loss of control, I am experimenting with some implementation behavior in which there really is a fixed owner for a Pod. It would rely on an out-of-band provisioning/deprovisioning model, which is hinted at in the text. It also relates to authZ, but that's a separate discussion. Main point: it sounds like a separate Pod deprovisioning mechanism would be OK.

Reply:
Just to circle back on this discussion, I am 👍  on the text here, as it relates to `DELETE`